chpl/chpl-api/src/main/java/gov/healthit/chpl/ApiExceptionControllerAdvice.java

```diff
@@ -232,4 +242,12 @@ public ResponseEntity<ErrorResponse> exception(Exception e) {
                 HttpStatus.INTERNAL_SERVER_ERROR);
     }
 
+    @ExceptionHandler(RedisTimeoutException.class)
+    public ResponseEntity<ValidationErrorResponse> exception(RedisTimeoutException e) {
+        LOGGER.error(e.getMessage(), e);
+        return new ResponseEntity<ValidationErrorResponse>(ValidationErrorResponse.builder()
+                .errorMessages(SortedSets.immutable.of(errorMessageUtil.getMessage("redis.connection.timeout")))
+                .businessErrorMessages(SortedSets.immutable.of(errorMessageUtil.getMessage("redis.connection.timeout")))
+                .build(), HttpStatus.BAD_REQUEST);
```

Comment:
I'm not sure this should be a bad request? It seems like more of an Internal Server Error to me.